scripts/k8s/load-images-to-kind.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Print usage options for this script.
+print_help() {
+  cat <<EOF
+Usage: $(basename "${0}") --tag <TAG>
+
+Options:
+  -h, --help        Display this text.
+  --tag <TAG>       Input the container image tag.
+
+Examples:
+  $(basename "${0}") --tag "ribbit"
+EOF
+}
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/../.."
+# Imports
+source "$ROOT_DIR/scripts/utils/log.sh"
+
+set -e
+
+TAG=
+
+while test $# -gt 0; do
+  arg="$1"
+  case "$arg" in
+  --tag)
+    test $# -lt 2 && log_fatal "missing value for the argument '$arg'"
+    TAG=$2
+    shift
+    ;;
+  --tag=*)
+    TAG=${arg#*=}
+    ;;
+  -h* | --help*)
+    print_help
+    exit 0
+    ;;
+  *)
+    print_help
+    log_fatal "unexpected argument '$arg'" 1
+    ;;
+  esac
+  shift
+done
+
+if [ -z "$TAG" ]; then
+  log_fatal "requires an image tag"
+fi
+
+IMAGE_TAG="v${TAG#v}"
+images=("upgrade-job" "obs-callhome" "obs-callhome-stats" "metrics-exporter-io-engine")
```

Comment:
Can this be loaded from other places? ex: sourcing release.sh or something

Reply:
The exact image names are built using string concat on this file: [nix/pkgs/images/default.nix](https://github.com/openebs/mayastor-extensions/blob/develop/nix/pkgs/images/default.nix)

The release.sh contains the nix-build targets, which are not the container image namespace-name thing.
I could add in a TODO to generate this list from the nix file somehow for a later date, WDYT?

Reply:
Ah right, maybe this can be addressed with the oep I raised
yep add a todo for now, thanks